core/modules/search/search.install

```diff
@@ -201,6 +201,27 @@ function search_update_1001() {
   update_variable_del('search_logging');
 }
 
+/**
+ * Set all search blocks with 'default' title to 'custom' title with 'Search'.
+ */
+function search_update_1002() {
+  $config_names = config_get_names_with_prefix('layout.layout.');
+  foreach ($config_names as $config_file) {
+    $config = config($config_file);
+    $data = $config->get();
+    foreach ($data['content'] as $hash => $block) {
+      if ($block['plugin'] == 'search:form'
+          && $block['data']['settings']['title_display'] == 'default') {
```

Comment:
Not sure if such splitting of conditionals is according to our coding standards.

Reply:
While less preferable than using separate variables, I found that we have at least 45 instances of doing this in code currently. I think updating the coding standards is in order to include reasonable separation of conditions on different lines by `&&` or `||` operators.

Reply:
If i wanted to use separate variables, would it need to add 4 extra lines, like this?
```php
      $block_title_default = FALSE;
      if ($block['data']['settings']['title_display'] == 'default') {
        $block_title_default = TRUE;
      }
      if ($block['plugin'] == 'search:form' && $block_title_default) {
```

Or could we do use a ternary even though it's more than 80 characters?
```php
      $default = $block['data']['settings']['title_display'] == 'default'? TRUE: FALSE;
      if ($block['plugin'] == 'search:form' && $default) {
```

Reply:
One option is:

```php
$block_title_display = $block['data']['settings']['title_display'];
if ($block['plugin'] == 'search:form' && $block_title_display == 'default') {
```

However I tend to steer away from using a variable like this when it's just used in one place only. Not sure if that's right or wrong though...

> Or could we do use a ternary even though it's more than 80 characters?

Code can be longer than 80 characters, it's only comments that are affected by that restriction (to my knowledge).